webbpsf/trending.py

```diff
@@ -177,15 +203,21 @@ def wfe_histogram_plot(opdtable, start_date=None, end_date=None, thresh=None):
     opdtable0 = webbpsf.mast_wss.deduplicate_opd_table(opdtable)
     opdtable1 = webbpsf.mast_wss.filter_opd_table(opdtable0, start_time=start_date, end_time=end_date)
 
-    webbpsf.mast_wss.download_all_opds(opdtable1)
+    if download_opds:
+        webbpsf.mast_wss.download_all_opds(opdtable1)
 
     # Retrieve all RMSes, from the FITS headers. 
     # These are observatory WFE (OTE + NIRCam), at the WFS sensing field point
     rmses=[]
+
+    if download_opds is False:
+        rmses = opdtable['rms_wfe']
```

Comment:
This seems potentially confusing, since now the download_opds switch is controlling something completely unrelated to downloading. Maybe this should be a separate switch? Or a check on the contents of the table to see if the `rms_wfe` column exists, or something like that?

Reply:
Yeah I can change the switch to see if the column already exists in the opdtable.